src/components/map/toolbar/EventLog.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 import { observer } from 'mobx-react';
-import { FaTimes, FaExclamation } from 'react-icons/fa';
+import classnames from 'classnames';
+import { FaTimes, FaExclamation, FaUndo } from 'react-icons/fa';
 import { FiClipboard } from 'react-icons/fi';
+import ILogEntry from '~/models/ILogEntry';
+import logActions from '~/enums/logActions';
```

Comment:
logTypes? Okay here action word fits better. But how about logActionType? At least that is not a verb.

Reply:
Action is both a noun and a verb, and I think in this context it is clear that the word is a noun.